graph_box_umbrella/apps/graph_box_web/lib/graph_box_web/router.ex

```diff
@@ -2,27 +2,29 @@ defmodule GraphBoxWeb.Router do
   use GraphBoxWeb, :router
 
   pipeline :browser do
-    plug :accepts, ["html"]
-    plug :fetch_session
-    plug :fetch_flash
-    plug :protect_from_forgery
-    plug :put_secure_browser_headers
+    plug(:accepts, ["html"])
+    plug(:fetch_session)
+    plug(:fetch_flash)
+    plug(:protect_from_forgery)
+    plug(:put_secure_browser_headers)
```

Comment:
Did Elixir ls add the parens to everything like this? How do we turn off?

Reply:
supposedly I could add 
```Elixir
locals_without_parens: [plug: 1, plug: 2],
```
But i'm not having any luck with it.

Reply:
I'm going to go ahead and merge this so that I can keep working off of it. We can swing back around to this.